### ISSUE TITLE
Can't take screenshot in sway

### ISSUE THREAD

**Issue by HarHarLinks** [thenaterhood/gscreenshot#123]
```shell
$ gscreenshot-cli -s
unknown output '/tmp/242193.png'No screenshot taken.

$ gscreenshot-cli -s -f gscreenshot.jpg
unknown output '/tmp/244465.png'No screenshot taken.

$ gscreenshot-cli -s -c
unknown output '/tmp/242285.png'No screenshot taken.
```

The gui does not seem to work either with similar messages when running from a shell.

Using arch linux. version info:
```shell
$ gscreenshot-cli -V
Using Grim screenshot backend
gscreenshot 2.16.1

$ echo $XDG_SESSION_TYPE
wayland
```
sway 1.6.1, grim 1.3.2, slurp 1.3.2, wl-clipboard 1:2.0.0-2

**Comment by thenaterhood** [comment]
Thanks for the report! I'll look into it.

**Comment by thenaterhood** [comment]
Should now be fixed in release 2.16.2
